### ISSUE TITLE
Error: 'POST param “code” doesn’t exist?

### ISSUE THREAD

**Issue by oadslug** [engram-design/FeedMe#91]
Getting an error when saving a feed entry with the latest craft build (Craft CMS 2.6.2930). Here is the error from craft.log (changed paths to hide server details). I'll try and track it down, but any ideas what might be causing this? Thanks — Douglas

```
******************************************************************************************************
2016/09/08 05:50:19 [error] [exception.Craft\HttpException.400] exception 'Craft\HttpException' with message 'POST param “code” doesn’t exist.' in /path_to_domain//craft/app/services/HttpRequestService.php:509
Stack trace:
#0 /path_to_domain//craft/app/controllers/UsersController.php(336): Craft\HttpRequestService->getRequiredPost('code')
#1 /path_to_domain//craft/app/framework/web/actions/CInlineAction.php(49): Craft\UsersController->actionSetPassword()
#2 /path_to_domain//craft/app/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#3 /path_to_domain//craft/app/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#4 /path_to_domain//craft/app/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#5 /path_to_domain//craft/app/framework/web/CWebApplication.php(282): CController->run('setpassword')
#6 /path_to_domain//craft/app/etc/web/WebApp.php(818): CWebApplication->runController('users/setpasswo...')
#7 /path_to_domain//craft/app/etc/web/WebApp.php(288): Craft\WebApp->_processActionRequest()
#8 /path_to_domain//craft/app/framework/base/CApplication.php(185): Craft\WebApp->processRequest()
#9 /path_to_domain//craft/app/index.php(62): CApplication->run()
#10 /path_to_domain//html/index.php(30): require_once('/nfs/c07/h05/mn...')
#11 {main}
REQUEST_URI=/admin/feedme/feeds/5
HTTP_REFERER=http://domain.org/admin/feedme/feeds/5
---

******************************************************************************************************
```

**Comment by engram-design** [comment]
This seems to be triggered by the `actionSetPassword` controller? There are no references to `getRequiredPost('code')` in Feed Me. Does this only happen on the Feed Me pages?

**Comment by oadslug** [comment]
So far. Yes. Entries save fine. And haven’t experienced any other issues. Although haven’t tested much with the latest update. — Douglas

**Comment by oadslug** [comment]
hmmm… just tested local and not getting the error. Only on the server. Maybe a server issue? Either php version incompatibility? Or not all the app files uploaded properly? Let me do some more testing… I’ll let you no results regardless. Thanks for your help. — Douglas

**Comment by oadslug** [comment]
Correction: I’ll let you know results regardless.

**Comment by oadslug** [comment]
I think I may have discovered the issue — seems the javascript and css files did not update properly when committed to the server due to Issue 32 (https://github.com/engram-design/FeedMe/issues/32). I deleted, committed, reinstalled, and committed again. All better. I should have suspected as I did notice some layout issues in the footer, and interaction issues with the select dropdowns. Thanks again for your help. — Douglas
